hphp/util/asm-x64.h

```diff
@@ -1205,6 +1205,11 @@ struct X64Assembler {
     bytes(n, nops[n]);
   }
 
+  void emitExceptions(int n) {
+    while (n >= 2) ud2();
```

Comment:
That's right. Very odd. I will fix it and analyse why my tests were still working.